applications/tari_testnet_miner/src/test_utils/builders.rs

```diff
@@ -0,0 +1,342 @@
+// Copyright 2019. The Tari Project
```

Comment:
There must be a way to _not_ copy and paste this whole file

Reply:
The only way I can think of is too move all of out test code into a "test" crate
We can then import this as a dev dependancy

Reply:
That's one option. Possibly the best one.

Another is to have test-utils behind a feature flag.

Another is to move the miner module into tari_core.

C&P the test code is the _worst_ option. I suggest commenting out the test, deleting the copied test code; deciding on the right approach forward (on IRC / in an issue) and then re-enabling the test in another PR.

Reply:
This issue is not limited to just the miner. 

Option 2 will only work with a non test flag.